frameworks/MLNet/setup.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 HERE=$(dirname "$0")
-MLNET='mlnet'
+MLNET_PACKAGE='mlnet-linux-x64'
```

Comment:
Would it be possible to adjust this so it also works on ARM (and preferably also macos)? For example by storing the result of:
```python
import sys
import platform
platform = {"linux": "linux", "darwin": "osx"}.get(sys.platform)
machine =  {"arm64": "arm64", "x86_64": "x64"}.get(platform.machine())
print(f"mlnet-{platform}-{machine}")
```

Reply:
That should be in the above script already :)